docs/guide/introduction.md

```diff
@@ -97,4 +97,39 @@ The rest of the documentation will include many specific, practical examples of
 
 ## Content schemas
 
-Nulla vitae elit libero, a pharetra augue. Nullam quis risus eget urna mollis ornare vel eu leo. Maecenas sed diam eget risus varius blandit sit amet non magna. Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum nibh, ut fermentum massa justo sit amet risus. Praesent commodo cursus magna, vel scelerisque nisl consectetur et.
+The content field schema is another key concept that most Apostrophe development will use. The "schema" here simply refers to definition of fields allowed in a particular context (e.g., a page, piece, or widget type). This is important so that the user interface knows what fields to show editors. It also tells both the UI and server how to validate data for that context.
+
+Our blog post module might look something like this with its field schema:
```

Comment:
```suggestion
The field schema of a blog post might look something like this:
```

Reply:
"Our blog post module" refers to the example in the previous section. Also is to be a bit less dry.